posts/2024-05-07-24.0.0.5-beta.adoc

```diff
@@ -0,0 +1,554 @@
+
+To use this feature, add the following code to your `server.xml` file:
+
+[source,xml]
+----
+<featureManager>
+     <feature>expressionLanguage-6.0</feature>
+</featureManager>`
+----
+
+For more information, see the link:https://jakarta.ee/specifications/expression-language/6.0[Jakarta Expression Language 6.0 specification and Javadocs].
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27963
+// Contact/Reviewer: volosied,pnicolucci
+// // // // // // // //
+[#pages]
+== Jakarta Pages 4.0
+
+The pages-4.0 feature is an implementation of the Pages 4.0 Specification for Jakarta EE 11. The Pages 4.0 specification includes a couple of new features and the removal of previously deprecated functions.
+
+The `jakarta.servlet.jsp.ErrorData` class was updated to add support for the new `jakarta.servlet.error.method` and `jakarta.servlet.error.query_string` attributes.  The following deprecated classes, methods, and actions were removed:
+
+* `jakarta.servlet.jsp.JspException.getRootCause()`
+* Classes in `jakarta.servlet.jsp.el`.
+* `isThreadSafe` page directive
+* `jsp:plugin` action and related actions
+* `jakarta.servet.jsp.tagext.BodyTag.EVAL_BODY_TAG` constant
+* Any methods that implemented `jakarta.el.ELResolver.getFeatureDescriptors()` were removed as the `getFeatureDescriptors()` method was removed in Expression Language 6.0.
+
+For more information and the change history of the specification between Pages 3.1 and Pages 4.0, see link:https://github.com/jakartaee/pages/blob/master/spec/src/main/asciidoc/ServerPages.adoc#c1-changes-between-jsp-40-and-jsp-31[Changes between 3.1 and 4.0].
+
+To use this feature, add the following code to your `server.xml` file:
+
+[source,xml]
+----
+<featureManager>
+     <feature>pages-4.0</feature>
+</featureManager>`
+----
+
+For more information, see the https://jakarta.ee/specifications/pages/4.0/[Jakarta Pages 4.0 Specification].
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27962
+// Contact/Reviewer: pmd1nh,pnicolucci
+// // // // // // // //
+[#servlet]
+== Jakarta Servlet 6.1
+
+The Open Liberty `servlet-6.1` feature is an implementation of the Servlet 6.1 specification for Jakarta EE 11 . It includes a number of new features, specification clarifications, and deprecates some existing servlet features.`
+

```

Comment:
Remove the extra ` at the end. (i.e existing servlet features.` )

Reply:
fixed